.spelling

```diff
@@ -381,6 +383,7 @@ OpenShift
 OpenSSL
 openssl
 OpenTracing
+ORIGINAL\_DST\_LB
```

Comment:
This shouldn't be in the dictionary. Use backticks around the word at the point of use.